<launch>
  <node pkg="giskard_examples" type="pr2_controller" name="pr2_controller" output="screen">
    <rosparam param="joint_names"
      file="$(find giskard_examples)/config/pr2_upper_body.yaml" />
    <param name="controller_description" 
      textfile="$(find giskard_examples)/controller_specs/pr2_upper_body_qp_position_control.yaml" />
    <param name="frame_id" value="base_link" />
    <remap from="~joint_states" to="/joint_states" />
  </node>

  <node pkg="giskard_examples" type="controller_action_server" name="controller_action_server" 
        output="screen">
    <remap from="~command" to="pr2_controller/goal" />
    <remap from="~current_goal_hash" to="pr2_controller/current_goal_hash" />
    <remap from="~feedback" to="pr2_controller/feedback" />
  </node>
</launch>
